requirements.txt

```diff
@@ -8,4 +8,6 @@ pygeobase>=0.3.15
 pynetcf>=0.1.17
 ascat>=1.0
 ismn>=0.3
-configparser
\ No newline at end of file
+configparser
+pykdtree
+xarray
```

Comment:
is xarray used without the new results manager?

Reply:
no, removed it